README.md

```diff
@@ -128,7 +128,15 @@ This is an example of how to list things you need to use the software and how to
    ```sh
    venv\Scripts\activate
    ```
-3. Create a posgresql database
+3. Install pre-commit
+   ```sh
+   pip install pre-commit
```

Comment:
The step to install requirements involves installing pre-commit. So we could put `pre-commit install` after the `pip install -r` step and maintain the number of steps.

Reply:
To clarify, `pre-commit` is in our dev requirements file.